scripts/generate-test-services.ts

```diff
@@ -42,7 +42,7 @@ const generatorConfigOpenApi: GeneratorOptions = {
   input: path.resolve('test-resources', 'openapi-service-specs'),
   outputDir: path.resolve('test-packages', 'test-services', 'openapi'),
   clearOutputDir: true,
-  generateJs: true,
```

Comment:
Hm, I enabled this to see if also the transpiling in other words correct code is generated. I think since the objects are used in so many tests we can be sure that it generate valide code. We also have the E2E tests.

Reply:
We generate the services twice, once in the core unit tests and once in the test services package. The unit tests should not contain any transpiled code, but the e2e tests should. At least that way it would be consistent with OData. (And that is the reason for this PR). I hope this also answers the second question ;)